src/Views/sidebarview.html

```diff
@@ -171,7 +171,7 @@ <h5>Supported search strings:</h5>
                             <div class="col-md-3 "><button class="link-btn" ng-click="vm.showScenarioHelp = !vm.showScenarioHelp"><i class="fa fa-question-circle"></i></button></div>
                         </div>
 
-                        <div ng-init="vm.showScenarioHelp = false" ng-show="vm.showScenarioHelp">
+                        <div ng-="vm.showScenarioHelp = false" ng-show="vm.showScenarioHelp">

```

Comment:
Maybe this was an accident?

```suggestion
                        <div ng-init="vm.showScenarioHelp = false" ng-show="vm.showScenarioHelp">

```